delivery_carrier_service_level/models/stock_picking.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Therp BV <https://therp.nl>
```

Comment:
Nitpicking: put a dot at the end of the line, like on the license line. Just more consistent. Not blocking.